### PR TITLE
Merge prometheus tags into metric tags (fixes #1)

### LINKED ISSUE #1: Pass OpenTSDB tags to Prometheus tags
Hello,

your exporter works well, thank you so much.
It would be nice if it possible to pass tags from OpenTSDB to prometheus.

For example:
```
          "subQuery": {
            "metric": "my.openttsdb.example.metric",
            "aggregator": "sum",
            "rate": false,
            "downsample": "10s-avg-none",
            "tags": {
              "deviceid": "*"
            }
          }
```
returns:
```
  {
    "metric": "my.openttsdb.example.metric",
    "tags": {
      "deviceid": "AABBCCDD",
      "tableid": "0"
    },
    "aggregateTags": [],
    "dps": {
      "1563192620": 20,
      "1563193220": 20,
      "1563193820": 20,
      "1563194480": 20,
      "1563195080": 20,
      "1563195740": 20
    }
  }

```
pass all returned tags to prometheus tags:
`my_openttsdb_example_metric{deviceid=AABBCCDD, tableid=0} 19`

### PR DESCRIPTION
**Pull request by krupenik**

### CODE REVIEW

File: web/test/MetricsControllerSpec.scala [hunk 1]
```diff
@@ -76,12 +76,12 @@ class MetricsControllerSpec extends PlaySpec
 
         val output = contentAsString(result).split("\n").toList
         output must contain theSameElementsInOrderAs List(
-          "# HELP test_app_metrics_one TestApp metrics: one",
-          "# TYPE test_app_metrics_one counter",
-          "test_app_metrics_one{severity=critical,escalation=pagerduty} 15",
           "# HELP test_app_metrics_two TestApp metrics: two",
           "# TYPE test_app_metrics_two counter",
-          "test_app_metrics_two{severity=critical,escalation=pagerduty} 10"
+          "test_app_metrics_two{severity=critical,escalation=pagerduty,role=developer} 10",
```
Comment (andreifecioru) [inline web/test/MetricsControllerSpec.scala]: This was a bug on my side. I fixed that in the latest commit (please take a look). Solving the problem here is fine as a quick fix, but it does not solve the underlying problem (it only masks the problem).

The issue was the way the `MetricsRepoService` is loading the metrics configuration file (the ordering of the files is unpredictable). I fixed that by making sure that the list of files being loaded is sorted alphabetically.

With that fix in place, we can rely in the unit-tests on a predictable/reliable ordering of the metrics which makes these changes unnecessary.

So, my suggestion is to rebase on top of master, drop the changes in the unit-tests and re-submit the PR.

**Review by andreifecioru — CHANGES_REQUESTED** [review]
Please rebase on master (changes in unit-tests should no longer be necessary). Leave only the changes related to the merging of tags.

File: web/test/MetricsControllerSpec.scala (hunk 1 above)
Comment (krupenik) [reply web/test/MetricsControllerSpec.scala]: thanks for the feedback and quick fix. PR rebased and updated.

**Review by andreifecioru — APPROVED** [review]
:ok:

**Pull request merged**